src/app/_components/UserListClient.tsx

```diff
@@ -0,0 +1,21 @@
+'use client';
+import { trpc } from '../_trpc/client';
+import { serverClient } from '../_trpc/serverClient';
```

Comment:
Consider renaming `trpc` and `serverClient` here. It doesn't seem the current naming here describes their roles.